prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -109,10 +109,29 @@ periodics:
 # kyma-integration-cleaner removes all sshPublic keys stored for service account "sa-vm-kyma-integration". Those keys refers to machines that in most cases were already removed.
 # Purpose of this job is to avoid error: "Login profile size exceeds 32 KiB" when running kyma-integration tests
 - name: utilities-kyma-integration-cleaner
-  cron: "0 7 * * 1-5" # “At 07:00 on every day-of-week from Monday through Friday.”
+  cron: "0 7 * * 1-5" # "At 07:00 on every day-of-week from Monday through Friday."
   labels:
     preset-sa-vm-kyma-integration: "true"
     preset-gc-project-env: "true"
   spec:
     containers:
     - image: eu.gcr.io/kyma-project/prow/cleaner:0.0.1 # see test-infra/prow/images/cleaner
+- name: orphanded-disks-cleaner
+  cron: "0 * * * *" # "Every hour"
```

Comment:
Are you sure every hour is necessary? @aszecowka @michal-hudy what do you think, how often should it run?

Reply:
Do you think it's too often? What's the suggested value then?

Reply:
I would start with once a day and go from there increasing the frequency as required

Reply:
I personally prefer to remove it ASAP, after all, we pay for these resources. It means they should be removed once they're not needed. One hour is not particularly fast but is fast enough to cut the costs down.